.github/actions/paths-filter/action.yml

```diff
@@ -163,6 +163,16 @@ outputs:
         github.event_name == 'push' &&
         steps.filter-special-branch.outputs.stable-branch == 'true'
       }}
+  rdbms-integration-tests:
+    description: Output whether RDBMS integrations tests should be run based on code changes
+    value: >-
+      ${{
+        github.event_name == 'push' ||
+        github.event_name == 'pull_request' ||
```

Comment:
By default we should use `== 'push' || ... == 'schedule'` here to account for the merge queue, and avoid `== 'pull_request'` to skip execution if nothing changed on a PR

Reply:
Mental model: this expression controls whether the tests will be run. and we want to run them "when needed" aka when something relevant to them changed, _always_ on `push` (to the `main` branch) and always on `schedule` (since we want to repopulate caches and build everything).